services/antivirus/pkg/service/service.go

```diff
@@ -144,6 +145,7 @@ func (av Antivirus) Run() error {
 // process the scan
 func (av Antivirus) process(ev events.StartPostprocessingStep) (scanners.ScanResult, error) {
 	if ev.Filesize == 0 || (0 < av.m && av.m < ev.Filesize) {
+		av.l.Debug().Str("uploadid", ev.UploadID).Uint64("limit", av.m).Uint64("filesize", ev.Filesize).Msg("Skipping file because file size is higher than the limit.")
```

Comment:
Suggestion:
`Skipping file...` --> `Skipping file to be virus scanned because its file size is higher than the defined limit.`

Reply:
```suggestion
		av.l.Info().Str("uploadid", ev.UploadID).Uint64("limit", av.m).Uint64("filesize", ev.Filesize).Msg("Skipping file to be virus scanned because its file size is higher than the defined limit.")
```
This is not expected behaviour better to `Info` users about it. (included Martins suggestion)